src/configure/configure.ts

```diff
@@ -88,7 +116,7 @@ class PipelineConfigurer {
         if (this.inputs.isNewOrganization) {
             this.inputs.project = {
                 id: "",
-                name:  generateDevOpsProjectName(this.inputs.sourceRepository.repositoryName)
+                name: generateDevOpsProjectName(this.inputs.sourceRepository.repositoryName)
```

Comment:
you might want to log telemetry that  new org/project was created

Reply:
we are logging it in the getAzureDevOpsDetails function